relayer-cli/src/commands/query/clients.rs

```diff
@@ -14,8 +16,27 @@ use crate::prelude::*;
 /// Query clients command
 #[derive(Clone, Command, Debug, Options)]
 pub struct QueryAllClientsCmd {
-    #[options(free, help = "identifier of the chain to query")]
+    #[options(free, required, help = "identifier of the chain to query")]
     chain_id: ChainId,
+
+    #[options(
+        help = "filter for clients which target a specific chain id",
+        meta = "ID"
+    )]
+    src_chain_id: Option<ChainId>,
+
+    #[options(
+        help = "omit printing the chain identifier which each client targets; \
+                providing a `-s`/`--src-chain-id` will enable this option",
+        default = "false"
```

Comment:
Not sure if we can shorten this a bit. `hermes help query clients` is hard to read.

Reply:
I adjusted the help messages a bit to make them more succint.

34c341111abb665df4a6b304adc9b04e51f0ee34

Reply:
looks good!